static/js/enketo/widgets/notewidget.js

```diff
@@ -63,6 +69,24 @@ define( function( require, exports, module ) {
         /* jshint unused:false */
     };
 
+    function applyLiveLinkHtml( $el ) {
+        var html = $el.html();
+
+        html = html.replace( /\[([^\]]*)\]\(([^)]*<[^>]*\>[^)]*)\)/gm,
+                '<a class="live-link" href="#" target="_blank">$1<span class="href">$2</span></a>' );
```

Comment:
Since I don't see a reason to ever display the hidden bit, I'd add a class `hidden` (or whatever class we have that already exists for hiding things) to it, and remove the custom CSS in other file.